incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/openapi/model/PathItem.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.command.config.internal.openapi.model;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class PathItem
+{
+    private Map<String, Operation> methods;
+
+    public Operation get;
+    public Operation put;
+    public Operation post;
+    public Operation delete;
+    public Operation options;
+    public Operation head;
+    public Operation patch;
+    public Operation trace;
+
+    public void initMethods()
+    {
+        methods = new LinkedHashMap<>();
+        if (get != null)
+        {
+            methods.put("GET", get);
+        }
```

Comment:
Suggest hoisting this to a method.
```
private static void putIfNotNull(
    Map<String, Operation> methods,
    String method,
    Operation operation)
{
    if (operation != null)
    {
        methods.put(method, operation);
    }
}
```
Then `initMethods` reads a good bit simpler.
```
public void initMethods()
{
    methods = new LinkedHashMap<>();
    putIfNotNull(methods, "GET", get);
    putIfNotNull(methods, "PUT", put);
    ...
}
```

It actually feels as if this logic is the beginning of the intermediate model we discussed, and therefore doesn't really belong directly on the OpenAPI model.

Reply:
Yep, introducing an intermediate model would help to keep the OpenAPI model super simple/clean and let us deal with the required transformations/conversions in another class. I think this is a good idea, but I couldn't come up with a good name for it 😅. For now I created the package named `model2` and the class `PathItem2`.  Pls chk.

I will keep thinking about some better naming pattern, pls let me know if you have any suggestions.

I created `Server2` for the same reason, based on your review item about the `Server` class.